client/src/components/organisms/RequestForm.tsx

```diff
@@ -73,8 +74,14 @@ const RequestGroupForm: FunctionComponent<Props> = (props: Props) => {
     }
   }`
 
-  const [createRequest] = useMutation(createRequestMutation);
-  const [updateRequest] = useMutation(updateRequestMutation);
+  const [createRequest] = useMutation(createRequestMutation, {
+    onCompleted: () => { props.onSubmitComplete() },
+    onError: (error) => { console.log(error) }
```

Comment:
Do we want to maybe display something like "Unfortunately we could not display the updated information" on an error rather than console logging? @guy-ro @meganniu

Reply:
LOL sounds like a design decision. I'll msg the designers

Reply:
might be something we add in a diff release